### DISCUSSION TITLE
xrepo在进行库下载时出错

### DISCUSSION THREAD

**Discussion by alphaDeng** [xmake-io/xmake#2346] [Q&A] [answered]
在尝试使用xrepo下载库时的时候出现错误：

PS C:\Windows\System32\WindowsPowerShell\v1.0> xrepo install boost
updating repositories .. error: fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

error: execv(xmake require -j 24 --extra={system=false} boost) failed(-1)

但xrepo在一开始下载git的时候是能够下载下来的，可能是仓库的ssh密钥的问题？

**Comment by waruqi** [comment, marked as answer]
删下 `%localappdata%/.xmake/repositories` 目录后再试试

**Reply by alphaDeng** [reply]
可以了，非常感谢！

**Reply by StarBeat** [reply]
这种莫名其妙的错误，属实难顶
